src/blocks/ImageLinkGrid/Component.tsx

```diff
@@ -0,0 +1,49 @@
+import { cn } from '@/utilities/ui'
+
+import type { ImageLinkGrid as ImageLinkGridProps } from '@/payload-types'
+
+import { ImageMedia } from '@/components/Media/ImageMedia'
+
+type Props = ImageLinkGridProps & {
+  className?: string
+  imgClassName?: string
+}
+
+export const ImageLinkGrid = (props: Props) => {
+  const { columns, className, imgClassName } = props
+  const numOfCols = columns?.length ?? 1
+
+  const colsClasses: { [key: number]: string } = {
+    1: 'sm:col-span-12',
+    2: 'sm:col-span-6',
+    3: 'sm:col-span-4',
+    4: 'sm:col-span-3',
+  }
+  const colsSpanClass = colsClasses[numOfCols]
+
+  return (
+    <div className="container">
+      <div className="grid sm:grid-cols-12 gap-y-2 gap-x-2">
+        {columns &&
+          columns?.length > 0 &&
+          columns.map((col, index) => {
+            const { caption, image, link } = col
+            const lastOddElement = numOfCols % 2 && index === numOfCols - 1
+            return (
+              <div className={cn(`relative ${colsSpanClass} ${className}}`)} key={index}>
```

Comment:
I think passing in `className` like this will technically still merge tailwind class names correctly but if it's undefined we'll see that string in the classes (not really a big deal). But there is an extra `}` here. 

![rendered classes](<CleanShot 2025-05-05 at 08.34.03.png>)

```suggestion
              <div className={cn('relative', colsSpanClass, className)} key={index}>
```